randovania/games/cave_story/logic_database/Sand Zone.txt

```diff
@@ -121,37 +125,48 @@ Extra - starting_script: <TRA0010:0099:0036:0033
       Any of the following:
           Can Break Blocks
           Bubbler and Break blocks with Bubbler (Beginner)
-          Missiles ≥ 3 and Break blocks with Missiles (Intermediate)
-          Missiles ≥ 2 and Break blocks with Missiles (Advanced)
+          Missiles ≥ 3 and Missile Launcher and Break blocks with Missiles (Intermediate)
+          Missiles ≥ 2 and Super Missile Launcher and Break blocks with Missiles (Advanced)
   > Pickup (Polish Spot)
       Any of the following:
           Can Break Blocks
           Bubbler and Break blocks with Bubbler (Beginner)
-          Missiles ≥ 10 and Break blocks with Missiles (Intermediate)
-          Missiles ≥ 5 and Break blocks with Missiles (Advanced)
+          All of the following:
+              Missile Launcher or Super Missile Launcher
+              Any of the following:
+                  Missiles ≥ 10 and Break blocks with Missiles (Intermediate)
+                  Missiles ≥ 5 and Break blocks with Missiles (Advanced)
   > Before Omega
       Any of the following:
           Can Break Blocks
           Bubbler and Break blocks with Bubbler (Beginner)
-          Missiles ≥ 27 and Break blocks with Missiles (Intermediate)
-          Missiles ≥ 13 and Break blocks with Missiles (Advanced)
-          Missiles and Break blocks with Missiles (Expert)
+          All of the following:
+              Missile Launcher or Super Missile Launcher
+              Any of the following:
+                  Missiles ≥ 27 and Break blocks with Missiles (Intermediate)
+                  Missiles ≥ 13 and Break blocks with Missiles (Advanced)
+                  Missiles and Break blocks with Missiles (Expert) and Can Kill Bosses
```

Comment:
this specific one is worth a comment mentioning that you're farming missile drops